src/main/java/org/opentripplanner/netex/support/stoptime/AbstractStopTimeAdaptor.java

```diff
@@ -0,0 +1,91 @@
+package org.opentripplanner.netex.support.stoptime;
+
+import java.math.BigInteger;
+import java.time.LocalTime;
+import org.rutebanken.netex.model.TimetabledPassingTime;
+
+/**
+ * Wrapper around {@link TimetabledPassingTime} that provides a simpler interface for passing times
+ * comparison. Passing times are exposed as seconds since midnight, taking into account the day
+ * offset.
+ * <p>
+ * This class does not take Daylight Saving Time transitions into account, this is an error and
```

Comment:
@t2gran This comment applies to mapping, not validating TimeTabledPassingTimes. Provided TimeTabledPassingTimes should always be increasing along the journey, including on DST transition days.
Comments to be moved to StopTimesMapper?